Source/Scene/Cesium3DTileset.js

```diff
@@ -1995,6 +2037,7 @@ define([
             replacementList.remove(currentNode);
 
             --stats.numberContentReady;
+            totalMemoryUsage = tileset._statistics.textureMemorySizeInBytes + tileset._statistics.vertexMemorySizeInBytes + tileset._statistics.batchTableMemorySizeInBytes;
```

Comment:
Should we add a prototype function to `tileset` as `tileset.totalMemoryUsage()`, which can be used externally?